src/components/tables/UsersTable.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <q-table
+    class="shadow-5"
+    table-header-class="bg-grey-3"
+    hide-bottom
+    row-key="objectId"
+    :pagination="pagination"
+    :columns="columns"
+    :rows="users"
+    data-cy="users_table"
+  />
+</template>
+
+<script setup>
+import { onMounted, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import * as UsersService from 'src/services/UserService';
+
+const { t } = useI18n();
+const pagination = ref({
+  rowsPerPage: 0, // infinite
+});
+const columns = ref([{
+  name: 'name',
+  required: true,
+  label: t('UsersTable.text.nameColumn'),
+  align: 'left',
+  field: 'firstname',
+}, {
+  name: 'userName',
+  required: true,
+  label: t('UsersTable.text.userNameColumn'),
+  align: 'left',
+  field: 'username',
+}, {
+  name: 'email',
+  required: true,
+  label: t('UsersTable.text.emailColumn'),
+  align: 'left',
+  field: 'email',
+}]);
+const users = ref([]);
+
+onMounted(async () => {
+  await UsersService.find().then((data) => {
```

Comment:
put this in search function like libraries.

Reply:
done